yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1625,7 +1602,17 @@ func generateAssessmentReportHtml(reportDir string) error {
 		// marking this as empty to not display this in html report for PG
 		assessmentReport.SchemaSummary.SchemaNames = []string{}
 	}
-	err = tmpl.Execute(file, assessmentReport)
+
+	type CombinedStruct struct {
+		AssessmentReport
+		MigrationComplexityCategorySummary []MigrationComplexityCategorySummary
```

Comment:
Can't we put this in the `AssessmentReport` struct and with that we can also have this summary in the JSON report as well?